transaction/core/test-utils/src/lib.rs

```diff
@@ -38,6 +40,14 @@ pub fn create_ledger() -> LedgerDB {
     LedgerDB::open(path).unwrap()
 }
 
+pub fn outputs_comparer_normal(a: &TxOut, b: &TxOut) -> core::cmp::Ordering {
+    a.public_key.cmp(&b.public_key)
+}
+
+pub fn outputs_comparer_inverse(a: &TxOut, b: &TxOut) -> core::cmp::Ordering {
+    b.public_key.cmp(&a.public_key)
+}
```

Comment:
perhaps forward_sort_txouts and inverse_sort_txouts to keep in convention of naming functions as verbs

Reply:
changed to traits now, fixed